mscore/importmidi/importmidi_model.cpp

```diff
@@ -505,7 +505,7 @@ void TracksModel::reset(const MidiOperations::Opers &opers,
                         {
                         }
                   QString headerName() const override { return QCoreApplication::translate(
-                                          "MIDI import operations", "2x less\nmeasure count"); }
+                                          "MIDI import operations", "Halving the\nmeasure count"); }
```

Comment:
Then the `\n` won't be necessary I guess?

Reply:
It was needed for the shorter string before, so surely is still needed